optuna/multi_objective/study.py

```diff
@@ -373,6 +375,7 @@ def set_user_attr(self, key: str, value: Any) -> None:
 
         self._study.set_user_attr(key, value)
 
+    @deprecated_func("3.1.0", "6.0.0")
```

Comment:
nit: We can omit the deprecation message here since multi_objective module itself is deprecated.
```suggestion
```